src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
+type Shape = ('triangle' | 'circle' | 'rectangle');
+type Color = ('red' | 'green' | 'blue');
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
will be better to use enum for this values (in this way you'll have less chance to make an error)